### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by caylahamann**
quick PR to update the readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,21 +74,16 @@ WARRANTY OR DEDICATED SUPPORT. That said, we welcome feedback, contributions and
...
+- **Community forum**: We also encourage you to bring your experiences and questions to the [Explorers
 Hub](https://discuss.newrelic.com/tag/victoryvisualizations), where our community members collaborate on
 solutions and new ideas. New Relic hosts and moderates this online forum where customers can interact with
 New Relic employees as well as other customers to get help and share best
 practices. Like all official New Relic open source projects, there's a related
 Community topic in the New Relic Explorers Hub. You can find this project's
 topic/threads here: https://discuss.newrelic.com/tag/victoryvisualizations
 
-### Contributing to the Victory charting library
-
```
Comment (JuliaNocera) [inline README.md:102]: I'm assuming getting rid of these headers was suggested in a previous review. If that's the case then all looks good on this PR to me :)

File: README.md [hunk 2]
```diff
@@ -56,6 +44,7 @@ You'll get the best possible data out of this application if you also:
    ```bash
    nr1 nerdpack:clone -r https://github.com/newrelic/nr1-victory-visualizations.git
    cd nr1-victory-visualizations
+   npm install
```
Comment (JuliaNocera) [inline README.md:47]: this triggered a thought for me, do we want to add an .nvmrc file? Not as part of this PR but as a separate task?

**Review by JuliaNocera — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (caylahamann) [reply README.md:47]: Yes!!!

File: README.md (hunk 1 above)
Comment (caylahamann) [reply README.md:102]: Yes! It's supposed to be contributions for if you want to add to the app and support for if you have questions or feature requests. I think there was some previous confusion on that but this brings it more in line with what the typical open source documentation looks like (ex: https://github.com/newrelic/open-source-tools/blob/master/nerdpacks/oss-template/README.md)

**Pull request merged**